TFLite/src/TensorToEdge.py

```diff
@@ -0,0 +1,29 @@
+from common import Edge
+from tflite import TensorType
+import inspect
+
+#Module to convert Tensor to Edge
+class TensorToEdge:
+    
+    def __init__(self):
+
+        # Dictionary to map enum value to enum name
+        typeclass = TensorType.TensorType
+        self._tensor_type = dict()
+        for member in inspect.getmembers(typeclass):
+            if not member[0].startswith('_'):
+                if not inspect.ismethod(member[1]):
+                    self._tensor_type[member[1]] = member[0]
+
+    def convert(self, tensor):
+
+        # Creating edge and extracting features
+        new_edge = Edge.Edge(label = tensor.Name(), value = tensor)
+        new_edge.type = self._tensor_type[tensor.Type()]
+
+        shape = list()
+        for i in range(tensor.ShapeLength()):
+            shape.append(tensor.Shape(i))
+        new_edge.shape = shape
```

Comment:
Just a thought, will we be able to query base on dimensions of tensors? Basically, there are use cases when we want to find out how many > 4D tensors are present in the model. If not, then see if that can be added.

Reply:
Shape stored as ARRAY in spanner, it provides ARRAY functions for querying like ARRAY_LENGTH which will correspond to number of dimensions